tools/accuracy_checker/configs/README.md

```diff
@@ -0,0 +1,95 @@
+# How to use predefined configuration files
+
+## Structure
+
+Configuration file declares validation process. Every model has to have entry in `models` list. Each entry has to contain distinct `name`, `launchers` and `datasets` sections.
+
+Example:
+
+```yaml
+models:
+  - name: model_name
+
+    launchers:
+      - framework: dlsdk
+        adapter: adapter_name
+
+    datasets:
+      - name: dataset_name
+```
+
+Also there are composite models, each of which has to have entry in `evaluations` list. Each entry has to contain distinct `name`, `module` and `module_config`. `module_config` has to consist of `network_info`,`launchers` and `datasets` fields.
+
+Example:
+
+```yaml
+evaluations:
+  - name: model_name
+    module: name_of_class_with_custom_evaluators
+    module_config:
+      network_info:
+        encoder: {}
+
+        decoder:
+          adapter: adapter_name
+
+      launchers:
+        - framework: dlsdk
+
+      datasets:
+        - name: dataset_name
+```
+
+## Location
+
+Predefined configuration file `accuracy-check.yml` for each Open Model Zoo model can be found in the model directory.
+
+`model_name.yml` file, which is located in current `configs` folder, is a link to `accuracy-check.yml` for `model_name` model.
```

Comment:
since it is a variable, for clearance better use <>
```suggestion
`<model_name>.yml` file, which is located in current `configs` folder, is a link to `accuracy-check.yml` for `<model_name>` model.
```